tests/Jenkins/dirac_ci.sh

```diff
@@ -368,7 +368,9 @@ function miniInstallDIRAC(){
 
   finalCleanup
 
-  #basic install, with only the CS (and ComponentMonitoring) running, together with DB InstalledComponentsDB, which is needed)
+  killRunsv
```

Comment:
why ? Also that will probably not work, as it's ran as `dirac` user, while `runsv` runs as root

Reply:
This is a bad name: it's not killing runsv, but the processes started by runsv. This is just a cleanup procedure.